adapters/repos/db/shard_write_delete.go

```diff
@@ -53,6 +54,27 @@ func (s *Shard) deleteObject(ctx context.Context, id strfmt.UUID) error {
 		return errors.Wrap(err, "get existing doc id from object binary")
 	}
 
+	obj, err := storobj.FromBinary(existing)
+	if err != nil {
+		return errors.Wrap(err, "get existing object from binary")
+	}
+
+	properties := obj.Object.Properties
+
+	// Remove each property from the property length tracker.
+	for _, prop := range properties.(map[string]interface{}) {
+		switch prop.(type) {
+		case string:
```

Comment:
isn't it `text` now?

Reply:
done

Reply:
afaik string does not exist anymore, pinging @aliszka to confirm?